manifests/c/Canon/IJPrinterAssistantTool/1.90.3/Canon.IJPrinterAssistantTool.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Canon.IJPrinterAssistantTool
+PackageVersion: 1.90.3
+InstallerType: portable

```

Comment:
Based on the screenshot Stephen shared, at the very least, this needs to be `exe`